pytest_fixtures/api_fixtures.py

```diff
@@ -19,6 +19,19 @@
 
 # Global Satellite Entities
 
+if not settings.configured:
+    settings.configure()
+
+
+@pytest.fixture(scope='module')
```

Comment:
session scope?

Reply:
Yeah ! I should do it for all the default fixtures!